### PR TITLE
[IMP] open_academy: Tabs and search view T#53764

### PR DESCRIPTION
**Pull request by payen000**
(edited to reflect suggested changes)
Done! I deleted the empty page, updated my commit messages and used the placeholder attribute :)
image

Search view:
image

### CODE REVIEW

**Comment by payen000** [comment]
@Jerudiel @rolandojduartem

Could you review my code, please? :)

**Comment by Jerudiel** [comment]
+1 looks good!!

**Review by rolandojduartem — APPROVED** [review]
+1 nice!

**Comment by payen000** [comment]
@frahikLV Got +1 from teammates, could you review my PR please? :)

File: open_academy/views/course.xml [hunk 1]
```diff
@@ -5,10 +5,14 @@
...
                 <sheet>
-                    <group>
-                        <field name='title'/>
-                        <field name='description'/>
-                    </group>
+                    <h1><field name='title'/></h1>
+                    <notebook>
+                        <page string='description'>
+                            <p>Esta es la descripción del curso <field name='title'/>:</p>
+                            <p><field name='description'/></p>
```
Comment (frahikLV) [inline open_academy/views/course.xml:12]: I recommend the use of `placeholder` attribute for this case, also, all the code need be written on English.

```suggestion
                            <field name="description" placeholder="This is the description of the course"/>
```
This it's going to look at this

image

This is more clean and easy to mantain and translate.

File: open_academy/views/course.xml [hunk 2]
```diff
@@ -5,10 +5,14 @@
...
+                        </page>
+                        <page string='syllabus'></page>
```
Comment (frahikLV) [inline open_academy/views/course.xml:14]: Don't include empty pages

**Review by frahikLV — CHANGES_REQUESTED** [review]
@payen000 can you address these changes?

Also, improve your commits message

I suggest to:
[IMP] open_academy: Tabs on form view T#53764
set as:
[IMP] open_academy: Include description tab to course form view T#53764

And
[IMP] open_academy: Search view (by title and description) T#53764
change to:
[IMP] open_academy: Allow search by title and description in the course model T#53764

**Comment by payen000** [comment]
@frahikLV This time it's done :) could you review, please?

**Review by frahikLV — APPROVED** [review]
@payen000 This LGTM 

I will merge this PR, perform rebasing on the branch you are working on today.

**Pull request merged**